### PR TITLE
Minor fixes branch

### PR DESCRIPTION
**Pull request by NastiaS**

### CODE REVIEW

File: src/components/DashboardPage/index.js [hunk 1]
```diff
@@ -28,11 +30,12 @@ import UpdateNotifier from 'components/UpdateNotifier'
 import BalanceInfos from 'components/BalanceSummary/BalanceInfos'
 import BalanceSummary from 'components/BalanceSummary'
 import Box from 'components/base/Box'
+import IconEmptyAccountTile from 'icons/illustrations/EmptyAccountTile'
 import PillsDaysCount from 'components/PillsDaysCount'
 import Text from 'components/base/Text'
 import OperationsList from 'components/OperationsList'
 import StickyBackToTop from 'components/StickyBackToTop'
-
+import Button from '../base/Button/index'
```
Comment (meriadec) [inline src/components/DashboardPage/index.js]: prefer using `components/base/Button'

File: src/icons/illustrations/EmptyAccountTile.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        <use fill="#FFF" xlinkHref="#f" />
+        <rect fill="#999" x="39" y="13" width="120" height="5" rx="2.5" />
+        <rect fill="#D8D8D8" x="39" y="23" width="70" height="5" rx="2.5" />
+        <g transform="translate(17 13)">
+          <use fill="#FCB653" fillRule="nonzero" xlinkHref="#g" />
+        </g>
+      </g>
+    </g>
+  </svg>
+)
```
Comment (meriadec) [inline src/icons/illustrations/EmptyAccountTile.js]: prefer put the `.svg` file in `static/images` and use it as an `` tag

File: src/components/DashboardPage/index.js (hunk 1 above)
Comment (gre) [reply src/components/DashboardPage/index.js]: autocomplete problem 🤣

Comment (NastiaS) [reply src/components/DashboardPage/index.js]: yea it was an autocomplete lol

**Pull request merged**